examples/nucleo_g474re/sh1106_i2c/main.cpp

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2023, Raphael Lehmann
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#include <modm/board.hpp>
+#include <modm/driver/display/sh1106_i2c.hpp>
+// #include <modm/driver/display/ssd1306_i2c.hpp>
+
+/// SH1106 display in I2C mode
+using MyI2cMaster = modm::platform::I2cMaster1;
+using Scl = Board::D15;
+using Sda = Board::D14;
+
+using Display = modm::Sh1106I2c<MyI2cMaster>;
+// using Display = modm::Ssd1306<MyI2cMaster>;
+
+Display display;
+
+int
+main()
+{
+	Board::initialize();
+
+	modm::delay(100ms);
+
+	MyI2cMaster::connect<Sda::Sda, Scl::Scl>();
+	// Use 20kHz I2C and internal pull-ups, works without external pull-ups
+	// if the jumper wires are short enought
+	MyI2cMaster::initialize<Board::SystemClock, 20_kHz>();
+	Sda::configure(Sda::InputType::PullUp);
+	Scl::configure(Sda::InputType::PullUp);
```

Comment:
The I2C master connect function takes an argument configuring the pull-ups. It defaults to `PullUps::External`. It would probably be cleaner to use that instead of calling `configure()` on the Gpios.

```suggestion
	// Use 20kHz I2C and internal pull-ups, works without external pull-ups
	// if the jumper wires are short enought
	MyI2cMaster::connect<Sda::Sda, Scl::Scl>(MyI2cMaster::PullUps::Internal);
	MyI2cMaster::initialize<Board::SystemClock, 20_kHz>();
```

Reply:
Ohh yeah, I remember...
Thanks!